build.sh

```diff
@@ -65,14 +66,20 @@ mkdir -p $OUTDIR/lib/bootstrap
 cp node_modules/bootstrap/dist/js/bootstrap.min.js $OUTDIR/lib/bootstrap/bootstrap.min.js
 cp node_modules/bootstrap/dist/css/bootstrap.min.css $OUTDIR/lib/bootstrap/bootstrap.min.css
 
+mkdir -p $OUTDIR/lib/streams
+cp node_modules/@openpgp/web-stream-tools/lib/*.js $OUTDIR/lib/streams
+# patch imports with .js, e.g. replace './streams' with './streams.js'
+# until https://github.com/openpgpjs/web-stream-tools/pull/20 is resolved
+sed -i -E "s/'\.\/(streams|util|writer|reader|node-conversions)'/'\.\/\1\.js'/g" $OUTDIR/lib/streams/*
```

Comment:
I can add a more complex regex to search for
`import { ... } from './file'`
and `import * as ... from './file'`

What do you think?

Reply:
It's a temporary fix, right? I think ok as is.